src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
-// write code here
+// {
+//   name,
+//   position,
+//   salary,
+//   age
+// }
+
+function sortList(list) {
+  return list.sort((a, b) => b.salary - a.salary);
+}
+
+function getEmployees(list) {
+  const objects = [];
+
+  list.forEach((e) => {
+    const employeeName = e.textContent.trim();
+    const position = e.getAttribute('data-position');
+    const salary = parseInt(
+      e.getAttribute('data-salary').replace('$', '').replace(',', ''),
+      10,
+    );
+    const age = e.getAttribute('data-age');
```

Comment:
Consider converting the `age` attribute to a number using `parseInt` or `Number` to ensure it is treated as a numerical value. This will be helpful if you need to perform any numerical operations on the age data.